ScoutSuite/providers/aws/resources/ec2/ami.py

```diff
@@ -14,6 +14,9 @@ async def fetch_all(self):
             self[name] = resource
 
     def _parse_image(self, raw_image):
-        raw_image['id'] = raw_image.get('ImageId')
-        raw_image['name'] = raw_image.get('Name')
+        raw_image['id'] = raw_image['ImageId']
+        raw_image['name'] = raw_image['Name']
+        if 'Tags' in raw_image:
```

Comment:
The same as my previous comment on lines 19 and 20 of this change

Reply:
Ah, yeah, I will remove this one.  Sorry about that.

Reply:
Sorry, I was still reviewing the changes. There are still changes related to tags in the following files:

- ScoutSuite/providers/aws/resources/ec2/instances.py
- ScoutSuite/providers/aws/resources/ec2/snapshots.py
- ScoutSuite/providers/aws/resources/ec2/volumes.py